tools/package.mjs

```diff
@@ -0,0 +1,57 @@
+
+import { readFileSync, writeFileSync } from 'fs';
+import { join } from 'path';
+
+import { exit } from 'process';
+
+import { hideBin } from 'yargs/helpers';
+import yargs from 'yargs';
+
+function getArgs() {
+	const argv = yargs(hideBin(process.argv))
+		.version(false)
+		.option('version', { demand: true, alias: 'v', type: 'string', default: '0.0.0', description: 'Version' })
+		.option('output', { demand: false, alias: 'o', default: 'dist', description: 'Output directory' })
+		.option('root', { demand: false, alias: 'r', default: '.', description: 'Root directory' })
+		.argv;
+
+	return argv;
+}
+
+function generatePackage() {
+	const packageMetadata = (version, dependencies) => ({
+		name: 'jewel-case',
+		version,
+		author: 'TradingView, Inc.',
+		description: 'Desktop application destribution',
+		license: 'MIT',
+		keywords: ['msix', 'deb', 's3', 'dmg'],
+		repository: {
+			type: 'git',
+			url: 'https://github.com/tradingview/jewel-case',
+		},
+		main: 'index.js',
```

Comment:
`index.{js,cjs,mjs}` should be used by default (btw, now you should have `index.mjs` after compilation, haven't you?)

Reply:
Fixed by 2bfab2d81f94db923614638e504c04e12f6a9eaf.